### PR TITLE
Content(missions) : Unfettered intro + medium warship

### PR DESCRIPTION
**Pull request by Hurleveur**
**Content (Artwork / Missions)**
Disclaimer : these are the first missions I do (but don't hold back I want to know all the issues !)

## Summary
Added 4 new missions(one of them in two parts) for the Unfettered Hai, which serve as an introduction to the campaign I am planning. They require you to have sold different amounts of jump drives first, the "friendly" missions that just ask for people transport and then cargo only require one jump drive sold, whilst the others like fighting an Unfettered captain, or even going to a jump drive cache, require more trust/infamy and thus you need to have sold 2 and then 3 jump drives to the Unfettered.
hai sea scorpion
I added a new ship, that comes from the "Beccas-Endless-Expansion-1.2.0" plugin, it is a medium warship, the Sea Scorpion, used by the Unfettered challenger, meant to be a glass canon, as a missile boat or equipped with pulse. The Lighting Bug felt too pathetic for a captain in my story whilst the Water Bug did not have enough guns, and I did not want to give him a shield beetle either. The ship fits the personality of the character perfectly ! (added them to the Unfettered fleets and shipyard)

## Save File
This save file can be used to play through the new mission content:
[Emile Lancer~JD from the void.txt](https://github.com/endless-sky/endless-sky/files/7706773/Emile.Lancer.JD.from.the.void.txt)
The Shield beetle and the 4 jump drives you start with should be enough to do all the missions.

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: [195](https://github.com/endless-sky/endless-sky-high-dpi/pull/195)

### CODE REVIEW

**Comment by MasterOfGrey** [comment]
@beccabunny can you confirm use with permission for your assets here?

**Comment by Hurleveur** [comment]
> @beccabunny can you confirm use with permission for your assets here?

image

**Comment by beccabunny** [comment]
> @beccabunny can you confirm use with permission for your assets here?

Can confirm

File: data/hai/hai ships.txt [hunk 1]
```diff
@@ -411,6 +411,95 @@ ship "Lightning Bug" "Lightning Bug (Surveillance)"
...
+	engine -15 70 0.8
+	engine 15 70 0.8
+	leak "leak" 40 50
+	leak "flame" 40 30
+	explode "tiny explosion" 24
+	explode "small explosion" 50
+	explode "medium explosion" 30
+	explode "large explosion" 14
+	"final explode" "final explosion medium"
+	description "Most of the ships the Unfettered had back when they splitted from the rest of the Hai weren't usable as missile boats, and for a while they had to rely heavily on pulse weapons instead. Only after some decades of hard work they were able to take the Sea Scorpion into mass production, but it was worth it: its low price and high number of gun ports make it perfect for its role, at the cost of a weak hull and shielding."
```
Comment (EjoThims) [inline data/hai/hai ships.txt:469]:
```suggestion
	description "Most of the ships the Unfettered had when they splintered from the rest of the Hai weren't usable as missile boats, so they had to rely heavily on pulse weapons instead. Only after several decades of hard work on a unique design were they able to take the Sea Scorpion into mass production, but it was well worth it; its low price and high number of gun ports make it perfect for its role, even at the cost of its weak hull and shielding."
```

**Comment by Hurleveur** [comment]
I was hoping to get some feedback before I get act 1 of the campaign done but it looks like that wasn't really the case, so I will close this pr and open a new one, with the entire thing this time!

**Pull request closed without merging**